docs/python-unit-tests.md

```diff
@@ -94,8 +94,7 @@ These default environments are:
   versions of Python, Django, and Wagtail. We require this environment to
   pass in CI.
 
-Tests will run against the default Django database as configured in settings
-using the `DATABASE_URL` environment variable.
+Tests will run against the default Django database.
```

Comment:
Does this mean one needs to define `DATABASE_URL` in the environment to run `tox` against PostgreSQL running in Docker?

Reply:
Hm, good catch! To clarify, this means a mix of virtualenv for Python code and Postgres as a Docker container? In that case, wouldn't you already have to have manually set DATABASE_URL to something in your .env? If so, tox should already use that.

Reply:
> To clarify, this means a mix of virtualenv for Python code and Postgres as a Docker container? 

Correct.

> In that case, wouldn't you already have to have manually set DATABASE_URL to something in your .env? If so, tox should already use that.

Well, part of this PR's change gets rid of tox configuration that defaults to the containerized Postgres:
https://github.com/cfpb/consumerfinance.gov/pull/6756/files#diff-ef2cef9f88b4fe09ca3082140e67f5ad34fb65fb6e228f119d3812261ae51449L84

I wonder though... it feels weird to remove that line from tox and make it rely on the env var when we're switching to explicitly defining the URL in a bunch of other places.

Reply:
The other thing I notice is I said "defaults to containerized Postgres", but really, it just defaults to a Postgres that's running locally with a password.

Should we standardize on password/nopassword for both local Postgres and Docker Postgres? Then it won't matter which it is for tox, the same URL will work for either from localhost and it can use the Django settings default.

Reply:
> Should we standardize on password/nopassword for both local Postgres and Docker Postgres?

I may be misremembering, but I seem to recall that we had to set a password for the Docker Postgres. I don't feel strongly about this, but it's convenient not to use a password locally.

> the same URL will work for either from localhost and it can use the Django settings default

I don't exactly follow. By "either" here do you mean, Postgres running locally on your Mac vs. Postgres running in a container? if so, I don't see how the same URL could work.

Reply:
Right now the only difference between the URL settings default to in this PR and the URL I use to make tox use Docker Postgres is the password.

To use Docker Postgres and Elasticsearch from tox, here's the bottom of my .env:

```
if [ $(uname -s) = "Darwin" ]; then
  export DATABASE_URL=postgres://cfpb:${PGPASSWORD}@localhost/cfgov
  export ES_HOST=localhost
  export ES7_HOST=localhost
fi
```

Reply:
I've confirmed that if I remove the `POSTGRES_PASSWORD` variable from the `postgres` container in `docker-compose.yml`, and add `POSTGRES_HOST_AUTH_METHOD: trust`, I can use the default `postgres://cfpb@localhost/cfgov` with tox without setting a `DATABASE_URL` in the environment at all.

Reply:
So, this is the change that works for me:

```diff
    postgres:
        …
        environment:
            POSTGRES_USER: cfpb
            POSTGRES_PASSWORD: cfpb
            POSTGRES_DB: cfgov
+           POSTGRES_HOST_AUTH_METHOD: trust
        …
```

Then I can run tox from outside of Docker and it will use the default database URL you have from settings and use the `postgres` Docker container, all without a password. This also means you could run cf.gov locally in a venv and use the `postgres` Docker container too!

Reply:
I do wonder though if this would mean we'll need to define the `postgres` container in `docker-compose.prod.yml` without that env var.

Reply:
That's a good point @willbarton. After some further thought, I think it might be better to standardize on using the `cfpb` password everywhere. I will add a commit that proposes that, and also updates the docs.